server/pa.js

```diff
@@ -209,6 +209,32 @@ async function handleMessageClients(req, res) {
   }
 }
 
+const validateCheckDatabaseConnection = Validator.body(['braveKey', 'googleIdToken']).trim().notEmpty()
+
+async function handleCheckDatabaseConnection(req, res) {
+  const validationErrors = Validator.validationResult(req).formatWith(helpers.formatExpressValidationErrors)
+
+  if (validationErrors.isEmpty()) {
+    const braveAPIKey = req.body.braveKey
+
+    if (paApiKeys.includes(braveAPIKey)) {
+      try {
+        await db.getCurrentTime()
+        res.status(200).send({ message: 'success' })
+      } catch (err) {
+        helpers.logError(err)
+        res.status(500).send({ message: 'Error in Database Access' })
```

Comment:
What do you think about using status code `503` to be a little more specific?
Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#server_error_responses

Reply:
Yes I think that's a good idea, will update!